package.json

```diff
@@ -29,6 +29,7 @@
     "lib": "lib"
   },
   "dependencies": {
+    "mailgun-js": "^0.7.7",
```

Comment:
Yep, remove it from the dependencies. Add a note to the README (or make a page in the wiki). Apart from that, everything looks good. Don't worry about the error - it's only for node 0.8, which I guess mailgun does not support. I'll look at updating the node versions that the travis build runs.